tests/test.py

```diff
@@ -17,6 +17,14 @@ def test_datadep() -> None:
     assert data_dep.is_file(), f"{data_dep} does not exist"
     assert (size := data_dep.stat().st_size) == 591, f"{data_dep} has wrong size {size}"
 
+    data_dep = Path("tests/appimage_data_file.txt")
```

Comment:
I think it would be nice to put these in a separate test to make clear that this is not the data dep of the binary, but of the appimage. Make sure to call it in the list of tests at the bottom of this file.